### PR TITLE
Update ImageContentSourcePolicy yaml for non-GA LSO installs

### PR DESCRIPTION
**Pull request by clacroix12**
Fixes: #3441 

Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

**Review by shylesh — APPROVED** [review]
+1

File: ocs_ci/templates/ocs-deployment/local-storage-optional-operators.yaml [hunk 1]
```diff
@@ -6,11 +6,14 @@ metadata:
 spec:
   repositoryDigestMirrors:
   - mirrors:
-    - quay.io/openshift-qe-optional-operators
-    source: registry.redhat.io/openshift4
+    - brew.registry.redhat.io
```
Comment (vasukulkarni) [inline ocs_ci/templates/ocs-deployment/local-storage-optional-operators.yaml:9]: can you add a comment for this file to be used internally?

Comment (clacroix12) [reply ocs_ci/templates/ocs-deployment/local-storage-optional-operators.yaml:9]: @vasukulkarni do you have a comment in mind? I'm not sure what you mean

Comment (vasukulkarni) [reply ocs_ci/templates/ocs-deployment/local-storage-optional-operators.yaml:9]: @clacroix12 the one I have is `# This configuration should be used internally within redhat vpn ` on the top of the file

Comment (clacroix12) [reply ocs_ci/templates/ocs-deployment/local-storage-optional-operators.yaml:9]: My understanding is that this is reachable outside of the redhat network so I don't think we need that

Comment (vasukulkarni) [reply ocs_ci/templates/ocs-deployment/local-storage-optional-operators.yaml:9]: bit confusing atleast the last one resolves to psi address but if it works externally then we dont need comment for sure.

**Review by vasukulkarni — APPROVED** [review]

**Comment by clacroix12** [comment]
> PR validation
> 
> Cluster Name:
> Cluster Configuration: conf/ocsci/aws_i3_local_storage.yaml
> PR Test Suite: deployment
> PR Test Path: tests/
> Additional Test Params:
> OCP VERSION: 4.7
> OCS VERSION: 4.7
> 
> [Job FAILED](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/3675//testReport) (installation failed, tests not executed).

I saw this validation failure locally as well. I didn't think it was related to LSO but I didn't see this when deploying without LSO so I'm going to look into this issue.

**Comment by clacroix12** [comment]
@vavuthu it appears that one of your changes in [29605d34f](https://github.com/red-hat-storage/ocs-ci/commit/29605d34f0184ad51380f650acb886342df761c9) might be causing the verification failure here for LSO on AWS. I've created a ticket for the error I'm seeing [here](https://github.com/red-hat-storage/ocs-ci/issues/3644).

Given this issue isn't related to my changes and is an existing issue with LSO on AWS, as well as the fact that the LSO and OCS were both installed successfully I'm going to consider this PR verified.

**Review by petr-balogh — APPROVED** [review]
@clacroix12 can we maybe document somewhere how and where to get access to this registry if someone outside QE will want to use this unreleased LSO deployment?

**Comment by petr-balogh** [comment]
@clacroix12 can you please document the process how to get credentials for brew registry. Please open issue for that

**Comment by clacroix12** [comment]
@petr-balogh I created the documentation issue here: https://github.com/red-hat-storage/ocs-ci/issues/3664.

I'm going to go ahead and merge this PR.

**Comment by prsurve** [comment]
From the verification job I see LSO was using the 4.6 version actually, this has to be tested with the 4.7 LSO version and LSO install is failing with Error: ImagePullBackOff

**Comment by clacroix12** [comment]
> From the verification job I see LSO was using the 4.6 version actually, this has to be tested with the 4.7 LSO version and LSO install is failing with Error: ImagePullBackOff

Yeah, that's because of [this issue](https://github.com/red-hat-storage/ocs-ci/pull/3685) that I had to fix in order to ensure we were retrieving the PackageManifest from the correct CatalogSource. I explicitly tested and confirmed that the 4.7 LSO was installed when I was verifying that issue so we should be okay here. If you are still seeing ImagePullBackOff then you may want to verify that your pull-secret has been updated correctly with brew registry credentials.

**Comment by vavuthu** [comment]
> > From the verification job I see LSO was using the 4.6 version actually, this has to be tested with the 4.7 LSO version and LSO install is failing with Error: ImagePullBackOff
> 
> Yeah, that's because of [this issue](https://github.com/red-hat-storage/ocs-ci/pull/3685) that I had to fix in order to ensure we were retrieving the PackageManifest from the correct CatalogSource. I explicitly tested and confirmed that the 4.7 LSO was installed when I was verifying that issue so we should be okay here. If you are still seeing ImagePullBackOff then you may want to verify that your pull-secret has been updated correctly with brew registry credentials.

It works for 4.7 as well, for one of my PR I have verified LSO with 4.7 and it was successful.

job link: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/16436/consoleFull

**Pull request merged**